cmd/snap/cmd_connect_test.go

```diff
@@ -30,27 +30,19 @@ import (
 
 func (s *SnapSuite) TestConnectHelp(c *C) {
 	msg := `Usage:
-  snap.test [OPTIONS] connect <snap>:<plug> <snap>:<slot>
+  snap.test [OPTIONS] connect [<snap>:<plug>] [<snap>:<slot>]
```

Comment:
Yes, this is an akward behavior of go-flags... You can see it already with the existing implementation of disconnect.